datafusion/src/datasource/parquet.rs

```diff
@@ -439,10 +426,6 @@ mod tests {
             .fold(0, |acc, _| async move { acc + 1i32 })
             .await;
 
-        // test metadata
-        assert_eq!(table.statistics().num_rows, Some(8));
```

Comment:
These tests seem not to have been ported over the physical_plan/parquet.rs (or I may have missed them). It would be good to keep them

Reply:
Well spotted thanks!